src/t8_forest/t8_forest.c

```diff
@@ -1534,7 +1524,7 @@ t8_forest_write_vtk_ext (t8_forest_t forest,
 int
 t8_forest_write_vtk (t8_forest_t forest, const char *fileprefix)
 {
-  return t8_forest_write_vtk_ext (forest, fileprefix, 1, 1, 1, 1, 0, 0, 0, 0,
+  return t8_forest_write_vtk_ext (forest, fileprefix, 1, 1, 1, 1, 1, 0, 0, 0,
```

Comment:
Should `write_ghosts` really be enabled by default? For a quick look at a mesh, the ghost elements are mostly not needed.